From a705e622113ba192cdd6b71b482a50976d261805 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lothar=20Wa=C3=9Fmann?= <LW@KARO-electronics.de>
Date: Mon, 18 Mar 2019 15:52:42 +0100
Subject: [PATCH 5/8] soc: imx: select missing PM_GENERIC_DOMAINS for i.MX8
 systems

All i.MX8 systems require this config and currently only work, because
it is selected by other archs enabled from the defconfig.
Add the missing 'select' to be able to compile a kernel for just i.MX8
and not requiring some bogus other arch to be enabled also.
---
 drivers/soc/imx/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/imx/Kconfig b/drivers/soc/imx/Kconfig
index 3a1fa180f28f..5bf70c13afd6 100644
--- a/drivers/soc/imx/Kconfig
+++ b/drivers/soc/imx/Kconfig
@@ -3,6 +3,7 @@ menu "i.MX SoC drivers"
 menuconfig ARCH_MXC_ARM64
 	bool "NXP i.MX family"
 	depends on ARM64
+	select PM_GENERIC_DOMAINS
 	help
 	  Support for NXP MXC/iMX-based ARM64 family of processors
 
-- 
2.20.1

